ui/raidboss/data/05-shb/ultimate/the_epic_of_alexander.js

```diff
@@ -1239,7 +1250,7 @@ let getHeadmarkerId = (data, matches) => {
       alertText: {
         en: 'Far Tethers',
         de: 'Entfernte Verbindungen',
-        fr: 'Liens éloignés',
+        fr: 'Éloignez les liens',
```

Comment:
If the mechanics are to remove the links, my translation is better. If I didn't understand correctly because I watched the video, tell me. I may be wrong or have misunderstood.

Reply:
I think this is just inconsistent with the other few triggers that also say `Liens éloigné`.  Is there a reason to make this one different?

Reply:
My request is simple, for the trigger :
1: Is it just information to say the links are distant.
2: The mechanics is keep the links away.

For the others, it is an oversight on my part. According to your answer, either I modify the others, or I restore here.

Reply:
A thing that was pointed out to me is that this mechanic does not just show up on its own. There are multiple times and places during the encounter when the player must do a thing now, or later, or in a specific way.

Because of this, it is probably better to revert this to what it was before, where it simply tells the user "you have the far tether" rather than telling them *what they must do with it*.

Reply:
Ok, i restored